src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -1,6 +1,18 @@
 package mate.academy.service;
 
+import mate.academy.model.User;
+
 public class AuthenticationService {
+    private UserService userService = new UserService();
+
+    public boolean login(String email, String password) {
+        User user = userService.findByEmail(email);
+        if (user != null) {
+            return user.getPassword().equals(password);
+        }
+        return false;
+    }
```

Comment:
```suggestion
     return user != null && user.getPassword().equals(password);
    }
```
we can do it in one line